compiler/parser/src/fstring.rs

```diff
@@ -67,10 +67,18 @@ impl FStringParser {
                         Some('a') => ConversionFlag::Ascii,
                         Some('r') => ConversionFlag::Repr,
                         Some(_) => {
-                            return Err(InvalidConversionFlag);
+                            if expression[1..].trim().is_empty() {
+                                return Err(EmptyExpression);
+                            } else {
+                                return Err(InvalidConversionFlag);
+                            }
```

Comment:
```suggestion
                            return Err(if expression[1..].trim().is_empty() {
                                EmptyExpression
                            } else {
                                InvalidConversionFlag
                            };
```
This is also possible